Revit_Engine/Query/GetRevitParameterValue.cs

```diff
@@ -94,3 +94,4 @@ public static object GetRevitParameterValue(this IBHoMObject bHoMObject, string
     }
 }
 
+
```

Comment:
Why is a blank line added at the end of _every_ file? Will we have 10 blanks in 2022? 😨

Reply:
This one is still valid @FraserGreenroyd 🐒